examples/data-action-platform/apex/start-dap-job.apex

```diff
@@ -0,0 +1,22 @@
+// PREPARE INPUT
+plauti__dapApexApiV1.RunMacroJobOnRecordsInput input = new plauti__dapApexApiV1.RunMacroJobOnRecordsInput();
+input.flowInvokerReference = 'yourClassName'; // Shown in the dap record audit log to indicate what invoked the flow that executed the action
+input.dapMacroApiName = 'macro-youMacro'; // Your action's apiName prefixed with "macro-"
+input.recordIds = new List<Id>{
+	Id.valueOf('a0P17000005cSNgEAM'), 
+	Id.valueOf('a0P17000005cSNgEAM')
+}; // Id of the records that will be provided to your macro 
+// ^ The object type of these recordIds should be the same object type that is configured in the macro
+
+// RUN DAP ACTION ASYNCHRONOUSLY ON A COLLECTION OF RECORDS
+plauti__dapApexApiV1 dapApexApi = new plauti__dapApexApiV1();
+try {
+    plauti__dapApexApiV1.RunMacroJobOnRecordsOutput output = dapApexApi.runMacroJobOnRecords(input);
+	Id dapFlowId = output.dapFlowId; // Should be used to get the satus of the dap job
```

Comment:
*status

Reply:
```suggestion
	Id dapFlowId = output.dapFlowId; // Should be used to get the status of the dap job
```

Reply:
@ruudsnl would you like me to create a PR for this typo or is the added value to low?

Reply:
This is something customers will read right?